### PR TITLE
refactor: event 系统

### PR DESCRIPTION
**Pull request by Sruimeng**
目前存在两个个问题，大家看一下
1、老接口是否保留
2、点击/消息元素 是挂在 Player 上还是挂在各自的 合成上，现在为了方便操作，用户直接使用元素的事件emit更简单一些，不需要进行名称的筛选

@yiiqii  @wumaolinmaoan  @RGCHN  @liuxi150 
确定后更新 threejs的变动 及 单测

## Summary by CodeRabbit

- **New Features**
	- Introduced event emission for player activities, such as state changes and errors.
	- Enabled user interactions within the player component with dedicated event listeners.
	- Added new methods for controlling playback state and handling events in the player.
	- Improved interactivity within the rendering process through enhanced event bindings.

- **Improvements**
	- Enhanced event handling across the `Player` and related classes, promoting an event-driven architecture.
	- Deprecated outdated callback functions to streamline event management.
	- Refactored player creation and scene loading for improved modularity and clarity.
	- Updated data source references for enhanced flexibility in content management.

- **User Experience**
	- Improved responsiveness and interactivity in visual effects handling.
	- Enhanced navigation structure for better access to interactive content.
	- Updated resource loading process for more effective scene management.
	- Reduced user feedback for item interactions due to the removal of console logging on clicks.

### CODE REVIEW

File: packages/effects/src/player.ts [hunk 1]
```diff
@@ -266,7 +223,7 @@ export class Player implements Disposable, LostHandler, RestoreHandler {
     this.event = new EventSystem(this.canvas, !!notifyTouch);
     this.event.bindListeners();
     this.event.addEventListener(EVENT_TYPE_CLICK, this.handleClick);
-    this.interactive = interactive;
+    this.interactive = interactive ?? false;
```
Comment (yiiqii) [inline packages/effects/src/player.ts:226]: 多余

File: packages/effects/src/player.ts [hunk 2]
```diff
@@ -569,9 +525,9 @@ export class Player implements Disposable, LostHandler, RestoreHandler {
     if (!this.ticker || this.ticker?.getPaused()) {
       this.doTick(0, true);
     }
-    this.handlePlayableUpdate?.({
+    this.emit(EffectEventName.PLAYER_UPDATE, {
       player: this,
-      playing: false,
+      playing: true,
```
Comment (yiiqii) [inline packages/effects/src/player.ts:530]: check

File: packages/effects/src/player.ts [hunk 3]
```diff
@@ -605,9 +561,9 @@ export class Player implements Disposable, LostHandler, RestoreHandler {
     }
 
     this.ticker?.pause();
-    this.handlePlayableUpdate?.({
+    this.emit(EffectEventName.PLAYER_UPDATE, {
       player: this,
-      playing: false,
+      playing: true,
```
Comment (yiiqii) [inline packages/effects/src/player.ts:566]: check

**Review by yiiqii — CHANGES_REQUESTED** [review]

**Comment by zheeeng** [comment]
元素监听事件和合成监听事件应该都支持比较合理？

**Review by yiiqii — APPROVED** [review]

**Pull request merged**